HW2Question2Updatded.cpp

```diff
@@ -77,14 +108,11 @@ int main()
     treeNode *right = head->right;
     right->right = new treeNode(19);
     
-    if(checkNodeInTree(head, key1) && checkNodeInTree(head, key1) )
-    {
-        treeNode* answer = lowestCommonAncestor(head, key1, key2);
+    
+    treeNode* answer = lowestCommonAncestor(head, key1, key2);
+    if(answer != nullptr)
         cout<<"LCA = "<<answer->value<<endl;
```

Comment:
Maybe add a comment that on case of error the lowestCommonAncestor function already reported the problem.